src/renderer.tsx

```diff
@@ -63,6 +66,24 @@ try {
   console.error(`PastelDB.getDatabaseInstance error: ${error.message}`)
 }
 
+ipcRenderer.on(
```

Comment:
This looks like a lot of code to just get current environment.
Is there a reason why we can't use process.env.NODE_ENV for simplicity?

Reply:
because it already has a property so that's why we have used it to consistent https://www.electronjs.org/docs/api/app#appispackaged-readonly